src/coreclr/vm/jithelpers.cpp

```diff
@@ -3729,20 +3726,17 @@ HCIMPL1(void, JIT_CountProfile64, volatile LONG64* pCounter)
     LONG64 delta = 1;
     DWORD threshold = g_pConfig->TieredPGO_ScalableCountThreshold();
 
-    if (count > 0)
+    if (count >= (LONG64)(1 << threshold))
```

Comment:
```suggestion
    if (count >= (LONG64)(1LL << threshold))
```
Apparently I also need this for C++